modules/nf-core/ncbitools/vecscreen/main.nf

```diff
@@ -0,0 +1,46 @@
+process NCBITOOLS_VECSCREEN {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "sanger-tol/ascc_main:0.001-c1"
```

Comment:
I had a discussion with @eeaunin this afternoon. `biocontainers/ncbi-tools-bin:v6.1.20170106-6-deb_cv1` is the best choice but doesn't work because it misses the `procps` package. This `sanger-tol/ascc_main:0.001-c1` is one of our existing containers that contains vecscreen and works for this module. However we'll still try to add `procps` to `biocontainers/ncbi-tools-bin:v6.1.20170106-6-deb_cv1` to make things clearer.